packages/ibm-products/src/global/js/hooks/useOverflowItems.ts

```diff
@@ -85,10 +102,24 @@ export const useOverflowItems = <T extends Item>(
 
   const visibleItems = getVisibleItems();
   const hiddenItems = items.slice(visibleItems.length);
+  const firstItemKey: string = getMap()?.keys()?.next()?.value || '';
+  const firstItemWidth = getMap()?.get(firstItemKey) || 0;
```

Comment:
i'm slightly confused by what these variables are trying to accomplish. can you explain their purpose?